tests/client/crud_spec/framework.rs

```diff
@@ -298,7 +298,7 @@ macro_rules! check_coll {
         };
 
         let coll = match outcome_coll.name {
-            Some(ref str) => $db.collection(&str),
+            Some(ref s) => $db.collection(&s.clone()[..]),
```

Comment:
Cloning here feels a little weird. Is there a reason why this needed to be changed from from `&s` to `&s.clone()[..])`?